next.config.js

```diff
@@ -33,6 +33,11 @@ async function redirects() {
       destination: '/authentication/connect',
       permanent: true,
     },
+    {
+      source: '/authentication/connect',
+      destination: 'https://blockstack.github.io/stacks.js',
```

Comment:
@yknl would this be the best place to link to? My feeling is there should be a docs page instead of just references

Reply:
Agreed, we should have a usage guide to complement the tutorial.

Reply:
is that planned? I'd say we merge this tutorial during docs week and create a user guide issue to work on later

Reply:
https://github.com/blockstack/docs/issues/895

Reply:
yes sounds good to me